ArduPlane/Log.cpp

```diff
@@ -382,8 +425,10 @@ const struct LogStructure Plane::log_structure[] = {
 // @Field: Still: True when vehicle is not moving in any axis
 // @Field: Stage: Current stage of the flight
 // @Field: Hit: True if impact is detected
-    { LOG_STATUS_MSG, sizeof(log_Status),
-      "STAT", "QBfBBBBBB",  "TimeUS,isFlying,isFlyProb,Armed,Safety,Crash,Still,Stage,Hit", "s--------", "F--------" , true },
+// @Field: Bit: bitmask of vehicle state information
+// @FieldBitmaskEnum: Bit: LogStatus::Bit
+    { LOG_STATUS_MSG, sizeof(log_STAT),
+      "STAT", "QBfBBBBBBH",  "TimeUS,isFlying,isFlyProb,Armed,Safety,Crash,Still,Stage,Hit,Bit", "s---------", "F---------" , true },
```

Comment:
I think a 'Sup' field is a better approach

Reply:
Done